pycalphad/tests/test_model.py

```diff
@@ -35,7 +35,7 @@ def test_model_ne():
 
 def test_export_import():
     "Equivalence of Model using re-imported database."
-    test_model = Model(Database.from_string(ALNIPT_DBF.to_string(fmt='tdb'), fmt='tdb'), ['PT', 'NI', 'VA'], 'FCC_L12')
+    test_model = Model(Database.from_string(ALNIPT_DBF.to_string(fmt='tdb', if_incompatible='ignore'), fmt='tdb'), ['PT', 'NI', 'VA'], 'FCC_L12')
```

Comment:
Yes, this one does and so does one of the others in the `test_database.py`. You could change to pass warn or raise and see the warning for the names. Do you care to elaborate on what regressions you're expecting? If you are more comfortable with warn being the default, that's fine.